samples/azure-quantum/hello-world/README.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
This README file will get imported into docs.microsoft.com/samples, such that more description is likely needed here to help samples gallery users understand that these notebooks are for use with the AQ service. For example, changing the description to something like "Running your first job on the Azure Quantum service" may be helpful here.

Reply:
Fixed!